../include/afpfs-ng/libafpclient.h:23:46: error: unknown type name 'fd_set'
   23 |         int (*scan_extra_fds)(int command_fd,fd_set *set, int * max_fd);
      |                                              ^~~~~~

---

--- a/include/afpfs-ng/libafpclient.h
+++ b/include/afpfs-ng/libafpclient.h
@@ -4,6 +4,9 @@
 
 #include <unistd.h>
 #include <syslog.h>
+#ifndef __GLIBC__
+#include <sys/select.h>
+#endif /* musl */
 
 #define MAX_CLIENT_RESPONSE 2048
 
